File: src/main/java/com/destino/projeto_destino/dto/usuario/UsuarioDTO.java
package com.destino.projeto_destino.dto.usuario;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.util.Date;

public record UsuarioDTO(
        String id,
        String nome,
        String cpf,
        String email,
        String telefone,
        String perfil,
        String valido,

        @JsonFormat(pattern = "dd-MM-yyyy")
        Date cadastro
) {
}

File: src/main/kotlin/com/fatec/destino/dto/usuario/UsuarioDTO.kt
package com.fatec.destino.dto.usuario

import java.util.Date

data class UsuarioDTO (
    var id: String,
    var nome: String,
    var cpf: String?,
    var email: String,
    var telefone: String?,
    var perfil: String,
    var valido: String,
    var cadastro: Date?
)